website/routes.py

```diff
@@ -1124,6 +1142,31 @@ def make_url_map(app):
             '/project/<pid>/node/<nid>/get_registrations/',
         ], 'get', project_views.node.get_registrations, json_renderer),
 
+        # Draft Registrations
+        Rule([
+            '/project/<pid>/draft/',
+        ], 'get', project_views.drafts.get_draft_registrations, json_renderer),
+        Rule([
+            '/project/<pid>/draft/<draft_id>/',
+        ], 'get', project_views.drafts.get_draft_registration, json_renderer),
+        Rule([
+            '/project/<pid>/draft/',
+        ], 'post', project_views.drafts.create_draft_registration, json_renderer),
+        Rule([
+            '/project/<pid>/draft/<draft_id>/',
```

Comment:
I've collected the URLs added and removed by this PR into a Google Doc, and sent an email to BrianG and Jeff.